Loader/SymfonyFixturesLoader.php

```diff
@@ -72,6 +79,61 @@ protected function createFixture($class)
         return $this->loadedFixtures[$class];
     }
 
+
+    /**
+     * Returns the array of data fixtures to execute.
+     *
+     * @return array $fixtures
+     */
+    public function getFixtures()
+    {
+        $fixtures = parent::getFixtures();
+
+        if ($this->tag) {
+            $mapping = $this->fixtureTagMapping;
+            $tag = $this->tag;
+            $fixtures = array_filter(
+                $fixtures,
+                function ($fixture) use ($mapping, $tag) {
+                    return isset($mapping[$fixture]) && $mapping[$fixture] === $tag;
+                },
+                ARRAY_FILTER_USE_KEY
+            );
+        }
+
+        return $fixtures;
+    }
+
+    /**
+     * Creates an array of the fixtures and there tags
+     *
+     * @param array $services
+     *
+     * @return void
+     */
+    public function addFixtureTagMapping(array $services)
+    {
+        foreach ($services as $service => $tags) {
+            foreach ($tags as $attributes) {
+                if (key_exists('set', $attributes)) {
```

Comment:
Please use `array_key_exists` instead of `key_exists`.
`key_exists` is a deprecated php alias: https://bugs.php.net/bug.php?id=52806